source/includes/extracts-sharded-clusters-backup-restore-limitations.yaml

```diff
@@ -1,23 +1,20 @@
 ref: sharded-clusters-backup-restore-mongodump-mongorestore-restriction
 content: |
    
-   :binary:`~bin.mongodump` and :binary:`~bin.mongorestore`
-   :red:`cannot` be part of a backup strategy for 4.2+ sharded clusters
-   that have sharded transactions in progress, as backups created with
-   :binary:`~bin.mongodump` *do not maintain* the atomicity guarantees
-   of transactions across shards.
+   To use :program:`mongodump` and :program:`mongorestore` as a backup
+   strategy for sharded clusters, see :ref:`backup-sharded-dumps`.
 
-   For 4.2+ sharded clusters with in-progress sharded transactions, use
-   one of the following coordinated backup and restore processes which
-   *do maintain* the atomicity guarantees of transactions across shards:
+   Sharded clusters can also use one of the following coordinated backup
+   and restore processes, which maintain the atomicity guarantees of
+   transactions across shards:
```

Comment:
Would it make sense to also mention that these solutions provide these atomicity guarantees while still accepting writes?

Reply:
Updated!